department-of-veterans-affairs/caseflow
# Create CO Hearing Days in Caseflow

Resolves #8666 

### Description
Small utility class to create CO Hearing Days in production whenever Hearings Management Branch asks us to. Class has parameters encoded in it and need to be modified whenever the utility class is to be executed.

### Acceptance Criteria

### Testing Plan
1. Go to rails console
2. Execute the following command: GenerateCOHearingDaysSchedule.generate_schedule(Date.new(2019, 4, 1), Date.new(2019, 9,30))
3. Verify Hearing Days were created in your local environment for the dates specified in the method signature.
4. Verify the days created follow the criteria specified in the ticket associated with this PR.
